### PR TITLE
Feat node cli

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

File: templates/cli/lib/commands/generic.js.twig [hunk 1]
```diff
@@ -77,19 +77,19 @@ const client = new Command("client")
...
         }
 
-        if (key) {
+        if (key !== undefined) {
             globalConfig.setKey(key)
         }
 
-        if (projectId) {
+        if (projectId !== undefined) {
             localConfig.setProject(projectId, '');
```
Comment (eldadfux) [inline templates/cli/lib/commands/generic.js.twig:85]: What is the second parameter used for?

File: templates/cli/lib/sdks.js.twig [hunk 2]
```diff
@@ -61,19 +61,19 @@ const sdkForProject = async () => {
     .setSelfSigned(selfSigned)
     .setLocale("en-US");
 
-  if (key) {
+  if (cookie) {
     return client
 {% if sdk.test != "true" %}
-      .setKey(key)
-      .setMode("default");
+      .setCookie(cookie)
```
Comment (eldadfux) [inline templates/cli/lib/sdks.js.twig:67]: Is this a new setting?

Comment (christyjacob4) [reply templates/cli/lib/sdks.js.twig:67]: No i just changed the order, to give higher preference to cookies that an API key

File: templates/cli/lib/commands/generic.js.twig (hunk 1 above)
Comment (christyjacob4) [reply templates/cli/lib/commands/generic.js.twig:85]: setProject(projectId, projectName)
    Its the project name

**Review by eldadfux — APPROVED** [review]

**Pull request merged**